development/build/scripts.js

```diff
@@ -714,26 +714,27 @@ function createFactoredBuild({
           case 'ui': {
             renderHtmlFile({
               htmlName: 'popup',
-              groupSet,
-              commonSet,
               browserPlatforms,
               applyLavaMoat,
             });
             renderHtmlFile({
               htmlName: 'notification',
-              groupSet,
-              commonSet,
               browserPlatforms,
               applyLavaMoat,
               isMMI: buildType === 'mmi',
             });
             renderHtmlFile({
               htmlName: 'home',
+              browserPlatforms,
+              applyLavaMoat,
+              isMMI: buildType === 'mmi',
+            });
+            renderJavaScriptLoader({
               groupSet,
               commonSet,
               browserPlatforms,
               applyLavaMoat,
```

Comment:
ok to remove the mmi line?

Reply:
Unfortunately not because of this code in the `home.html` file:


```
{{@if(it.isMMI)}}
    <title>MetaMask Institutional</title>
{{/if}}
```